lang/cs/Org.Apache.REEF.Client/Common/IDriverHttpEndpoint.cs

```diff
@@ -24,8 +25,21 @@ namespace Org.Apache.REEF.Client.Common
     [Unstable("0.13", "Working in progress for what to return after submit")]
     public interface IDriverHttpEndpoint
     {
+        /// <summary>
+        /// Get Uri result
```

Comment:
This was existing code. It actually returns http response result for a given uri as a web request. We can revisit it to either rename or as it is an Unstable, may drop later.